### PR TITLE
Unsubmitted annotations

### PR DESCRIPTION
**Pull request by lukesneeringer**
This PR merges the not-yet-submitted annotations into the `input-contract` branch (maintaining its status as the "ahead-of-final-approval" working area).

As all microgenerator authors will be affected by this, using the review assignments as a mechanism to allow everyone to ack this change.

### CODE REVIEW

File: google/api/resource.proto [hunk 1]
```diff
@@ -71,9 +100,41 @@ extend google.protobuf.FieldOptions {
   //       }];
   //     }
   //
-  // Only one of {`resource`, `resource_reference`} may be set.
+  // Only one of {`resource`, `resource_set`, `resource_reference`} may be set.
```
Comment (chrisdunelm) [inline google/api/resource.proto]: There is a corner-case where both `resource` and `resource_set` can be set together.
It's when a resource is initially just a Resource (ie not a set), then it needs changing to a ResourceSet. To maintain backward compatibility the `resource` should remain, and the `resource_set` added.
We may wish to ignore this case for now, especially as resource-sets are still up for discussion.

Comment (lukesneeringer) [reply google/api/resource.proto]: > I suspect I'm missing something obvious! But...
> This PR seems to remove quite a bit of stuff that is required. E.g. LRO `OperationData`, method-signature, field_behavor, etc...

This PR merges in `master`, and therefore incorporates #31.

Comment (lukesneeringer) [reply google/api/resource.proto]: > We may wish to ignore this case for now, especially as resource-sets are still up for discussion.

Yeah. Comments are easy to fix.

**Comment by noahdietz** [comment]
@chrisdunelm I believe this branch was based on master, which has the new form of these protos i.e. s/OperationData/OperationInfo/ and moving `FieldBehavior` to a different file. This PR cleans up the overlap between what we had (input-contract) and what we will have (master+g3-future)

**Pull request merged**